src/vs/base/test/node/glob.test.ts

```diff
@@ -810,7 +819,6 @@ suite('Glob', () => {
 		testOptimizationForBasenames({
 			'**/foo/**': true,
 			'{**/bar/**,**/baz/**}': true,
-			'**/bulb/**': false
 		}, ['foo', 'bar', 'baz'], [
```

Comment:
I do not understand why we need to remove what is passed as the argument. 
Should just the result be changed?

Reply:
Ok. I will rework it.